framework/auth/cas.py

```diff
@@ -149,6 +149,20 @@ def _parse_profile(self, raw):
             resp.attributes[attribute] = data['attributes'][attribute]
         return resp
 
+    def revoke_application_tokens(self, client_id, client_secret):
```

Comment:
I'd love to see more integration tests for the parts that rely on CAS, actually. I couldn't think of a meaningful test based on mocks, but discussion is heartily welcomed.

Issue: this function hits an endpoint on realCAS, which either succeeds or fails based on state stored in a service that is not used by travis test suite, or even some devs running locally.